src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/movements/ExternalMovementsSyncService.kt

```diff
@@ -420,7 +420,7 @@ class ExternalMovementsSyncService(
       retryMessage.body,
     ).also {
       telemetryClient.trackEvent(
-        "$TELEMETRY_PREFIX-application-mapping-created",
+        "$TELEMETRY_PREFIX-application-mapping-retry-created",
```

Comment:
Changed all the retry mapping telemetry to be explicit that we're only tracking the retries because this confused me for a while.